aws/resource_aws_workspaces_workspace_test.go

```diff
@@ -0,0 +1,440 @@
+		F:    testSweepWorkspaces,
+	})
+}
+
+func testSweepWorkspaces(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+	conn := client.(*AWSClient).workspacesconn
+
+	var errors error
+	input := &workspaces.DescribeWorkspacesInput{}
+	err = conn.DescribeWorkspacesPages(input, func(resp *workspaces.DescribeWorkspacesOutput, _ bool) bool {
+		for _, workspace := range resp.Workspaces {
+			err := workspaceDelete(aws.StringValue(workspace.WorkspaceId), conn)
+			if err != nil {
+				errors = multierror.Append(errors, err)
+			}
+
+		}
+		return true
+	})
+	if testSweepSkipSweepError(err) {
+		log.Printf("[WARN] Skipping workspaces sweep for %s: %s", region, err)
+		return errors // In case we have completed some pages, but had errors
+	}
+	if err != nil {
+		errors = multierror.Append(errors, fmt.Errorf("error listing workspaces: %s", err))
+	}
+
+	return errors
+}
+
+func TestAccAwsWorkspacesWorkspace_basic(t *testing.T) {
+	directoryResourceName := "aws_workspaces_directory.test"
+	bundleDataSourceName := "data.aws_workspaces_bundle.test"
+	resourceName := "aws_workspaces_workspace.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsWorkspacesWorkspaceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Destroy: false,
+				Config:  testAccWorkspacesWorkspaceConfigA(),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAwsWorkspacesWorkspaceExists(resourceName),
+					resource.TestCheckResourceAttrPair(resourceName, "bundle_id", bundleDataSourceName, "id"),
+					resource.TestCheckResourceAttrPair(resourceName, "directory_id", directoryResourceName, "id"),
+					resource.TestMatchResourceAttr(resourceName, "ip_address", regexp.MustCompile(`\d+\.\d+\.\d+\.\d+`)),
+					resource.TestCheckResourceAttr(resourceName, "state", workspaces.WorkspaceStateAvailable),
+					resource.TestCheckResourceAttr(resourceName, "root_volume_encryption_enabled", "false"),
+					resource.TestCheckResourceAttr(resourceName, "user_name", "Administrator"),
+					resource.TestCheckResourceAttr(resourceName, "volume_encryption_key", ""),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.0.compute_type_name", workspaces.ComputeValue),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.0.root_volume_size_gib", "80"),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.0.running_mode", workspaces.RunningModeAlwaysOn),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.0.running_mode_auto_stop_timeout_in_minutes", "0"),
+					resource.TestCheckResourceAttr(resourceName, "workspace_properties.0.user_volume_size_gib", "10"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "2"),
```

Comment:
can you remove the tags from thie basic config and create a seperate tags test based on https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#resource-tagging-acceptance-testing-implementation

Reply:
Sure, I'll do this, thank you for pointing this out 🙇